movies.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/estilos-index.css">
+    <link rel="stylesheet" href="css/estilos-movies.css">
+    <link rel="stylesheet" href="css/flexboxgrid.min.css">
+    <title>Películas</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div>
+                <nav class="row nav-fija">
+                        <ul class="row col-xs-12 col-sm-12 col-md-12 col-lg-12">
+                            <li class="start-xs start-sm start-md start-lg"><a id="enlace-logo" href="index.html"><img id="logo-cine" src="imagenes/logo-cine.png" alt="Logo CineOnline"></a></li>
+                            <li><h3 id="nombre-cine">CineOnline</h3></li>
+                            <li><a href="index.html">Inicio</a></li>
+                            <li><a href="movies.html">Películas</a></li>
+                            <li><a href="">Series</a></li>
+                            <li><a href="">Precios</a></li>
+                            <li><a href="">Contacto</a></li>
+                            <li class="end-lg"><a href="" id="acceso">Acceder</a></li> 
+                        </ul> 
+                </nav>    
+            </div>
+        </header>
+    </div>
+    
+    <div class="row">
+        <div class=" row seccion-uno col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <div class=" row col-xs-6 col-sm-6 col-md-6 col-lg-6">
+                <div class="seccion-dos">
+                    <div class="seccion1movies">
+                        <h1>Películas en tendencia</h1> 
+                    </div>
+                </div>
+            </div> 
+                    
+                
+            <div class="row col-xs-6 col-sm-6 col-md-6 col-lg-6">  
+                <div class="seccion-dos">
+                    <div class="seccion2movies">
+                        <a class="listapeliculas" href="movies-list.html">Ver lista<img  class="lista" src="imagenes/iconolista.png.png"></a> 
+                    </div> 
+                </div>           
+            </div>
+        </div>  
+    </div> 
+
+<div class="container">
+    <div class="row">
+        <div class="seccion-dos col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <div id="seccion3movies">
+                <section class="seccion3categorias-etiquetas">

```

Comment:
This element represents a block in the layout and it would be best to use the column classes to be displayed beside its sibling element, for this to work they both should be inside a container element with `row` class.

Don't mix layout elements with float elements. `#seccion3categorias-etiquetas` has a float property and the sibling element has grid CSS classes. Use grid classes to arrange layout elements. 

If elements are displayed with float properties inside a container each element should be the right percentage they are occupying. For example, if 2 elements are inside a container and one element is 35% width the other one should be 65%.

Element `#seccion3categorias-etiquetas` has a `section` tag with semantically represents a generic standalone section of a document but instead visually is an aside element because its content is only indirectly related to the document's main content. Reference [HTML Semantic Elements](https://www.w3schools.com/html/html5_semantic_elements.asp)